Control/Uart.cpp

```diff
@@ -0,0 +1,33 @@
+#include "Arduino.h"
+#include "Uart.h"
+
+Uart::Uart() : stringComplete(false) {
+}
+
+void Uart::initializeCommUart() {
+    Serial.begin(9600);
+}
```

Comment:
I was checking this [C++ naming convention of Google](https://google.github.io/styleguide/cppguide.html#Naming), in summary:

- Functions have to be camel case with capital at the beginning: `IntializeComm`.
- Variables have to be snake case: `in_char`

Reply:
Sure, I will create a new issue and link a branch to fix convention standards while I am away.